corehq/util/metrics/prometheus.py

```diff
@@ -21,17 +21,30 @@ class PrometheusMetrics(HqMetrics):
     def __init__(self):
         self._metrics = {}
 
+    @property
+    def accepted_gauge_params(self):
+        return ['multiprocess_mode']
+
     def _counter(self, name: str, value: float = 1, tags: Dict[str, str] = None, documentation: str = ''):
         """See https://prometheus.io/docs/concepts/metric_types/#counter"""
         try:
             self._get_metric(PCounter, name, tags, documentation).inc(value)
         except ValueError:
             pass
 
-    def _gauge(self, name: str, value: float, tags: Dict[str, str] = None, documentation: str = ''):
-        """See https://prometheus.io/docs/concepts/metric_types/#histogram"""
+    def _gauge(self, name: str, value: float, tags: Dict[str, str] = None, documentation: str = '', multiprocess_mode='all'):
+        """
+        See https://prometheus.io/docs/concepts/metric_types/#histogram
+
+        multiprocess_mode: can be one of below values
```

Comment:
Could be useful to use constants for these.

Reply:
+1

Reply:
Updated.